pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -358,3 +361,4 @@ func newOauthClient(tokenSource oauth2.TokenSource) (*http.Client, error) {
 
 	return oauth2.NewClient(oauth2.NoContext, tokenSource), nil
 }
+}
```

Comment:
@gmarek , are you mean it is same with [client, err := newOauthClient(tokenSource)](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/gce/gce.go#L179)?